### PR TITLE
New sh file for deploy adkmobile

### PR DESCRIPTION
**Pull request by Zeeerooo**
Refactor run.sh
New functions for deployment (start, stop, delete, update_app, run)
Changed name uwsgi-adk to server-adk
Update AdkintunMobile version

### CODE REVIEW

File: adkintunmobile/server/run.sh [hunk 1]
```diff
@@ -1,61 +1,146 @@
...
+        echo "$num"
+        echo "You must use -u, -p and -d!";
+        usage_run;
+    fi
+    
+    cd "$DIR"
+
+    # Give as parameter the database name, the user and the password. They must be the same in config.py
+    docker run --name postgres-adk -e POSTGRES_PASSWORD=$p -e POSTGRES_USER=$u -e POSTGRES_DB=$d --restart=unless-stopped -p 5432:5432 --log-opt max-size=50m -d postgres
+    until nc -z $(docker inspect --format='{{.NetworkSettings.IPAddress}}' postgres-adk) 5432
```
Comment (fmontoto) [inline adkintunmobile/server/run.sh:64]: There Isn't a better way to do this =( ??

**Comment by fmontoto** [comment]
It seems fine :) LGTM

File: adkintunmobile/server/run.sh (hunk 1 above)
Comment (Zeeerooo) [reply adkintunmobile/server/run.sh:64]: I thought it was a good way :(

Comment (fmontoto) [reply adkintunmobile/server/run.sh:64]: hahaha, it looks like you're right. Sorry. LGTM

**Comment by camilog** [comment]
LGTM

**Pull request merged**